packages/opentelemetry-api-metrics/src/NoopMeter.ts

```diff
@@ -158,23 +158,26 @@ export class NoopMetric<T> implements UnboundMetric<T> {
 
 export class NoopCounterMetric
   extends NoopMetric<BoundCounter>
-  implements Counter {
+  implements Counter
+{
   add(value: number, labels: Labels) {
     this.bind(labels).add(value);
   }
 }
 
 export class NoopValueRecorderMetric
   extends NoopMetric<BoundValueRecorder>
-  implements ValueRecorder {
+  implements ValueRecorder
```

Comment:
I think we should change the rule to keep things like this
```
implements ValueRecorder {
...
}
```

instead of 
```
implements ValueRecorder 
{
...
}
```

Reply:
Any idea what rule this was? The failure just says `prettier/prettier`

Reply:
this can be something around `https://eslint.org/docs/rules/brace-style` and differences between `1tbs` and `allman` maybe ?

Reply:
Turns out to be quite difficult and annoying to change the rule. It is introduced by `prettier/prettier` which is explicitly not configurable. `prettier/prettier` is introduced by our reliance on `gts`. I am trying to remove the `gts` rules but am having to set a lot of rules that `gts` brought in.

Reply:
Brace style is back to how it was before